eclass/distutils-r1.eclass

```diff
@@ -1815,16 +1815,15 @@ distutils-r1_run_phase() {
 	fi
 
 	# Set up build environment, bug #513664.
-	local -x AR=${AR} CC=${CC} CPP=${CPP} CXX=${CXX}
```

Comment:
I already NAK-d this, and the whole underlying code is changed in #35554.

Reply:
I'm not certain why you're pointing this out since I already clarified I don't intend to have distutils-r1 changes merged yet, *and* this PR predates yours *and* I reviewed your PR and generally agree with your approach.

Even though your approach was only motivated by my proposal. :)

tl;dr I already agree with you. No worries.